### PR TITLE
[FIX]fix a typing error

### PR DESCRIPTION
**Pull request by ChaeyeonSeo**

### CODE REVIEW

**Review by devquint — CHANGES_REQUESTED** [review]
`\` is for the markdown grammar escape.

```markdown
>= hello
```
shows

>= hello

with quote grammar.

```markdown
\>= hello
```
shows

\>= hello

escaping quote grammar.


I think we should change line 17 from `< greater than` to `\> greater than`, keeping line 19 to `\>=`. Or you just keep them(because most of the people can understand what it wanna say), and close this PR.

**Comment by ChaeyeonSeo** [comment]
Thanks for reviewing. I changed the code.

**Review by devquint — COMMENTED** [review]
Wouldn't it be more common to say `>` is `greater than` and `<` is `smaller than`? Usually if we say "X < Y", it called "X is smaller than Y", in aspect of X.

You might think differently, so let's leave it as it is and listen to other people's opinions.

**Comment by ChaeyeonSeo** [comment]
Yes, I agree. And they also wrote greater than or equal to to "\>=". I will fix it.

File: python-for-beginners/08 - Handling conditions/README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ else:
 [Comparison operators](https://docs.python.org/3/library/stdtypes.html#comparisons)
 
 - < less than
-- < greater than
+- \> greater than             # \(quote grammar) is added problem related to markdown grammar escape
```
Comment (Hyuntae-Jeong) [inline python-for-beginners/08 - Handling conditions/README.md]: Please give me feedback about added comment if it is appropriate or not.
This comment was added for users who might wonder why \ is added

Comment (devquint) [reply python-for-beginners/08 - Handling conditions/README.md]: That's not markdown commenting grammar.

**Comment by devquint** [comment]
Reminder: @Hyuntae-Jeong please check for the comment, and fix them.

File: python-for-beginners/08 - Handling conditions/README.md (hunk 1 above)
Comment (Hyuntae-Jeong) [reply python-for-beginners/08 - Handling conditions/README.md]: What should i correct them into? @devquint 
Not sure about what "\" does, so i added comment

Comment (devquint) [reply python-for-beginners/08 - Handling conditions/README.md]: It's a markdown file, so if you want to comment, follow the markdown grammar.
Maybe most of the people won't look at the raw markdown text.
If you still want to leave comments, google `markdown comment` and fix it to suitable grammar.

Comment (devquint) [reply python-for-beginners/08 - Handling conditions/README.md]: If you want to look what this REAMDME.md looks like, just press `view file` button at the `files changed` menu. [Direct link](https://github.com/20-1-SKKU-OSS/c9-python-getting-started/blob/33bf281b4a2cd615605e50b6ac85037244b39b12/python-for-beginners/08%20-%20Handling%20conditions/README.md)

Comment (Hyuntae-Jeong) [reply python-for-beginners/08 - Handling conditions/README.md]: Hmm.. I'll just delete it!

**Review by devquint — APPROVED** [review]

**Review by Dorika0203 — APPROVED** [review]

**Pull request merged**